tests/e2e/e2e.sh

```diff
@@ -17,83 +17,54 @@
 set -eux
 
 export ARTIFACTS="${ARTIFACTS:-$(mktemp -d)}"
+HUB="istio-testing"
+TAG="istio-testing"
+ROOT=$(cd ../../../; pwd)
 
-function cleanup_kind_cluster() {
-    kind export logs --name istio-testing "${ARTIFACTS}/kind"
-    kind delete cluster --name=istio-testing
-}
-
-function setup_kind_cluster() {
-  # Delete any previous e2e KinD cluster
-  echo "Deleting previous KinD cluster with name=istio-testing"
-  if ! (kind delete cluster --name=istio-testing) > /dev/null; then
-    echo "No existing kind cluster with name istio-testing. Continue..."
-  fi
-
-  trap cleanup_kind_cluster EXIT
-
-  # Create KinD cluster
-  if ! (kind create cluster --name=istio-testing --loglevel debug --retain --wait 30s); then
-    echo "Could not setup KinD environment. Something wrong with KinD setup. Exporting logs."
-    exit 1
-  fi
-
-  KUBECONFIG="$(kind get kubeconfig-path --name="istio-testing")"
-  export KUBECONFIG
-
-cat <<EOF > metallb-config.yaml
-apiVersion: v1
-kind: ConfigMap
-metadata:
-  namespace: metallb-system
-  name: config
-data:
-  config: |
-    address-pools:
-    - name: default
-      protocol: layer2
-      addresses:
-      - 172.17.255.1-172.17.255.250
-EOF
-
-  kubectl apply -f https://raw.githubusercontent.com/google/metallb/v0.8.1/manifests/metallb.yaml
-  kubectl apply -f ./metallb-config.yaml
-}
 
 function setup_docker() {
-  HUB=istio-testing TAG=latest make -f Makefile.core.mk controller docker
-  kind --loglevel debug --name istio-testing load docker-image istio-testing/operator:istio-testing
+  HUB=gcr.io/istio-testing TAG=1.5-dev make controller docker
```

Comment:
the hub should not be a real hub, or jts impossible to tell whether we are using the local image or accidentally using a published image

Reply:
Need to set this as a real hub otherwise we need to modify deploy/operator.yaml..

Reply:
I think it's better to modify that. Being able to change the hub/tag is a requirement for users, so we need to support it some way or another.

Otherwise, it's likely someone changes the image pull policy of something else that causes us to not use the local image and we don't notice?

Reply:
agree!  Let me put the operator.yaml one copy into testdata